### DISCUSSION TITLE
SD card crash

### DISCUSSION THREAD

**Discussion by aLAN-LDZ** [Jyers/Marlin#1407] [Q&A] [answered]
Hi

I have the latest version of your software installed. Everything was fine until today, I switched from the default to bltouch.

On the default software, I do not have this problem, on the bltouch software, when I click the first icon "print", the printer freezes and restarts.

when without an SD card, I first click on this icon and see the words no media and insert the card, I can see the files normally. Sometimes printing can be started.

I do not have this problem on the default software :( pls help

**Comment by Jyers** [comment, marked as answer]
Sounds like a bad SD card. Try a different one and see if that stops the issue.
